docs/upcoming_changes/README.rst

```diff
@@ -0,0 +1,40 @@
+
+Changelog
+=========
+
+This directory contains "news fragments" which are short files that contain a
+small **ReST**-formatted text that will be added to the next what's new page.
+
+Make sure to use full sentences with correct case and punctuation, and please
+try to use Sphinx intersphinx using backticks. The fragment should have a
+header line and an underline using ``========`` followed by description of
+your user-facing changes as they should appear in the relase notes.
+
+Each file should be named like ``<PULL REQUEST>.<TYPE>.rst``, where
+``<PULL REQUEST>`` is a pull request number, and ``<TYPE>`` is one of:
+
+* ``highlight``: Adds a highlight bullet point to use as a possibly highlight
+  of the release.
+* ``np_support``: Addition of new NumPy functionality.
+* ``deprecation``: Changes existing code to emit a DeprecationWarning.
+* ``future``: Changes existing code to emit a FutureWarning.
+* ``expired``: Removal of a deprecated part of the API.
+* ``compatibility``: A change which requires users to change code and is not
+  backwards compatible. (Not to be used for removal of deprecated features.)
+* ``cuda``: Changes in the CUDA API.
+* ``new_feature``: New user facing features like ``kwargs``.
+* ``improvement``: General improvements and edge-case changes which are
+  not new features or compatibility related.
+* ``performance``: Performance changes that should not affect other behaviour.
+* ``change``: Other changes
+
+If you are unsure what pull request type to use, don't hesitate to ask in your
+PR.
+
+You can install ``towncrier`` and run ``towncrier build --draft``
+if you want to get a preview of how your change will look in the final release
+notes.
+
+.. note::
+    This README was adapted from the NumPy changelog readme under the terms of
+    the MIT licence.
```

Comment:
I think that this should be a comment in a style compliant with https://github.com/numba/numba/blob/e7a1e8315d74b0f92ba8b26a40bbb1f28db94ba2/LICENSES.third-party#L454-L461
?